tests/wpt/web-platform-tests/html/semantics/embedded-content/the-img-element/img.complete.html

```diff
@@ -38,8 +38,11 @@
       assert_false(loaded);
       loaded = true;
       assert_true(document.getElementById("imgTestTag3").complete);
-      var currentSrc = document.getElementById("imgTestTag3").currentSrc
-      assert_equals(new URL(window.location.origin + "/" + currentSrc).pathname, "/3.jpg");
+      var currentSrc = document.getElementById("imgTestTag3").currentSrc;
+      var result = document.baseURI.split('/');
+      result.pop();
+      var expected_url = result.join('/')+"/3.jpg";
+      assert_equals(new URL(currentSrc).pathname, new URL(expected_url).pathname);
```

Comment:
Instead of playing around with URL components, let's do this instead:
```js
var currentSrc = document.getElementById("imgTestTag3").currentSrc;
var expectedUrl = new URL("3.jpg", window.location);
assert_equals(currentSrc.pathname, expectedUrl.pathname);
```

Reply:
Modifying the last line to as shown below, as currentSrc.pathname returns undefined.
``` 
assert_equals(new URL(currentSrc).pathname, expectedUrl.pathname);
```